### DISCUSSION TITLE
Strapi vs Payload - Developer experience

### DISCUSSION THREAD

**Discussion by JeffreyArts** [payloadcms/payload#7468] [General]
I'm a fan of Strapi, but the more in-depth I go. The harder it seems to be to figure out how to achieve stuff. Strapi's documentation is a true maze, and the Typescript implementation ain't great. I was curious if anyone here has had experience with Strapi, how they experienced that, and how their development experience is with PayloadCMS.

**Comment by smoothdvd** [comment]
Me too. So, I am trying PayloadCMS in a new project after nearly four years of developing with Strapi.

**Comment by davideperozzi** [comment]
Payloads DX is unmatched in the Headless CMS realm imo. I used Strapi for a couple of projects, but always had problems with wrong types, poor documentation and a confusing structure.

I mean in Strapi you can easily click all of your stuff without having to code really, but i don't know if that's a good thing, since I want migrations and manage all the collections and stuff via code and for that Payload ist way better with type completion, repeatable block management so on. Also way faster to develop, since in 3.0-beta you get HMR support even for the admin panel. Also writing plugins for Strapi is a pain + poorly documented. With Payload it's straight-forward (type autocompletion and such). 

Only downside right now for me is, they don't have "multi media upload" and folders for your media files, but they're working on it right now and I suppose it'll be released with 3.0 (stable). Still was worth the switch 100%!

If your project is not heavily depending on having folders in your media files and multi file uploads then you can safely use PayloadCMS  imho. You'll have a waaaay better DX than with Strapi.

**Comment by MarkAtOmniux** [comment]
I used Strapi for about a year, incredibly frustrating DevX and not veary easy to extend. I moved to payload in 2022 and haven't looked back since. We just migrated a website from Strapi over to PayloadCMS here - https://releese.io
